gluetool/help.py

```diff
@@ -324,3 +326,129 @@ def functions_help(functions):
     {{ body }}
     {% endfor %}
     """)).render(FUNCTIONS=[function_help(func, name=name) for name, func in functions])
+
+
+def extract_eval_context_info(source, logger=None):
```

Comment:
ok, so we actually need this just because we do not load any modules, so we need to parse the source right?

Reply:
Well, a module is loaded - otherwise, no one would be interested in its eval context, one has to load it to print its help -, it's just that it's not possible to get any variable declared inside its `eval_context` getter, since there's no stack frame we could inspect. The easiest way in is trough getter's source. I tried to use attribute of the property, to keep this information so I could just pick it from there (e.g. `eval_context.__content__ = ...` right after `eval_context` but at that moment it's `property` object and I was stopped :/ So it's the hard way, declaring a variable inside the body of `eval_context` getter.